Add info on rolling up uncompressed chunks (#1934)
timescale/docs: timescaledb/how-to-guides/compression/manually-compress-chunks.md

        now()::timestamp - INTERVAL '3 weeks'
    ) i;
```

> ## Roll up uncompressed chunks into compressed chunks

Reviewer: ```suggestion
## Roll up uncompressed chunks when compressing
```

> In TimescaleDB 2.9 and later, you can roll up multiple uncompressed chunks into
> a single, compressed, chunk as part of your compression procedure. This allows

Reviewer: ```suggestion
In TimescaleDB 2.9 and later, you can roll up multiple uncompressed chunks into
the previous compressed chunk as part of your compression procedure. This allows
```

> you to have much smaller uncompressed chunks. For example, if you have multiple

Reviewer: Slightly unclear. What is smaller here is the chunk time interval, not the number of bytes, but this can read either way. Probably good to be more clear on this.

> you to have much smaller uncompressed chunks. For example, if you have multiple

Reviewer: ```suggestion
you to have uncompressed chunks with a smaller chunk time interval than the compressed chunks. For example, if you have multiple
```

smaller uncompressed chunks in your data, you can roll them up into a single
compressed chunk. This allows you to have much smaller uncompressed chunks than
compressed chunks, which reduces your disk space use for uncompressed data.

To roll up your uncompressed chunks into a compressed chunk, use this command,
specifying the interval to use:

> ```sql

Reviewer 2: SELECT compress_chunk_time_interval('<time_interval>',if_not_compressed => true)
    FROM show_chunks(
        'example',
        now()::timestamp - INTERVAL '1 week',
    ) ;

compress_chunk_time_interval
```

The time interval you choose must be a multiple of the uncompressed chunk
interval. For example, if your uncompressed chunk interval is one week, your

> `compress_chunk_time_interval` could be two weeks, or six weeks, but not one

Reviewer 2: ```suggestion
`<time_interval>` of the compressed chunk could be two weeks, or six weeks, but not one
```

        now()::timestamp - INTERVAL '3 weeks'
    ) i;
```

## Roll up uncompressed chunks when compressing

In TimescaleDB 2.9 and later, you can roll up multiple uncompressed chunks into
a previously compressed chunk as part of your compression procedure. This allows
you to have uncompressed chunks with a smaller chunk time interval than the
compressed chunks. For example, if you have multiple smaller uncompressed chunks
in your data, you can roll them up into a single compressed chunk. This allows
you to have much smaller uncompressed chunk intervals, which reduces your disk

> space use for uncompressed data.

Reviewer 3: I find the two `This allows...` sentences repetitive in a confusing way. IMO the second is better because it explains the benefit, consider deleting the first or collapsing it into the second?

    ) i;
```

## Roll up uncompressed chunks when compressing

In TimescaleDB 2.9 and later, you can roll up multiple uncompressed chunks into
the previous compressed chunk as part of your compression procedure. This allows
you to have uncompressed chunks with a smaller chunk time interval than the compressed chunks. For example, if you have multiple
smaller uncompressed chunks in your data, you can roll them up into a single
compressed chunk. This allows you to have much smaller uncompressed chunks than

> compressed chunks, which reduces your disk space use for uncompressed data.

Reviewer: Some repetition in the text, otherwise looks good.
